### ISSUE TITLE
negative prob values for audio and dynamic_video

### ISSUE THREAD

**Issue by A-d-DASARE** [C-EXPR-DB/AVCER#11]
Hello, Thanks for open sourcing your code! why dynamic_dataframes despite being passed through softmax, it is giving negative scores? similarly with audio frames. And their probability is not summing up to 1 so they are logits I guess. It would be great if I get some response. Thanks!
